src/Utils/util.test.ts

```diff
@@ -70,4 +71,221 @@ describe('util', () => {
             expect(actual).toBe(true)
         })
     })
+
+    describe('isActionUniqueTEXT', () => {
+        it('exercise TEXT action creation', () => {
+            const sampleAction: CLM.ActionBase = {
+                actionId: "103726c8-2cfe-46e6-ad1e-fef8dbe4d6c3",
+                actionType: CLM.ActionTypes.TEXT,
+                createdDateTime: "2019-02-25T00:00:00.000Z",
+                isTerminal: true,
+                negativeConditions: [],
+                negativeEntities: [],
+                packageCreationId: 0,
+                packageDeletionId: 0,
+                payload: '{"json":{"kind":"value","document":{"kind":"document","data":{},"nodes":[{"kind":"block","type":"line","isVoid":false,"data":{},"nodes":[{"kind":"text","leaves":[{"kind":"leaf","text":"hi","marks":[]}]}]}]}}}"',
+                requiredConditions: [],
+                requiredEntities: [],
+                requiredEntitiesFromPayload: [],
+                suggestedEntity: null,
+                version: 0
+            }
+
+            let actionSet: CLM.ActionBase[] = []
+
+            test('as first action created', () => {
```

Comment:
Change these to be the similar language/syntax as the existing tests. They should be something like "given X return Y" or "when X should do Y"

Something similar to:
"given action not in list of actions return true"
"given action when identical action is in list return false"
...